omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -96,17 +86,67 @@ Configure OMERO.web and create the NGINX OMERO configuration file::
     omero config set omero.web.application_server wsgi-tcp
     omero web config nginx --http "${WEBPORT}" --servername "${WEBSERVER_NAME}" > /opt/omero/web/omero-web/nginx.conf.tmp
 
-For more customization, please read :ref:`customizing_your_omero_web_installation`.
+OMERO.web offers a number of configuration options.
+    The configuration changes **will not be applied** until
+    Gunicorn is restarted using ``omero web restart``.
+
+    -  Session engine:
+
+      -  OMERO.web offers alternative session backends to automatically delete stale data using the cache session store backend, see :djangodoc:`Django cached session documentation <topics/http/sessions/#using-cached-sessions>` for more details.
+
+      - `Redis <https://redis.io/>`_ requires `django-redis <https://github.com/jazzband/django-redis/>`_ in order to be used with OMERO.web. We assume that Redis has already been installed. Follow the step-by-step deployment guides from :doc:`../web-deployment`. To configure the cache, run::
```

Comment:
This references `sysadmins/unix/install-web/web-deployment.html` for information on redis but there's no mention of it there.